sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/mathExpressions.scala

```diff
@@ -603,8 +823,16 @@ case class Unhex(child: Expression) extends UnaryExpression with ImplicitCastInp
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
 @ExpressionDescription(
-  usage = "_FUNC_(x,y) - Returns the arc tangent2.",
-  extended = "> SELECT _FUNC_(0, 0);\n 0.0")
+  usage = "_FUNC_(expr1, expr2) - Returns the arc tangent2.",
```

Comment:
We should probably elaborate this a tiny bit, to say it's the arctangent of two arguments, because there's no "tangent2" function. Really it means (cribbing from wikipedia) the angle in radians between the positive x-axis of a plane and the point given by the coordinates (expr1, expr2) on it.